contracts/Core/parameters/child/AssetManagerParams.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+import "../interfaces/IAssetManagerParams.sol";
+import "./utils/GovernanceACL.sol";
+
+abstract contract AssetManagerParams is GovernanceACL, IAssetManagerParams {
+    uint16 public epochLength = 300;
+
+    function setEpochLength(uint16 _epochLength) external override onlyGovernance {
+        // slither-disable-next-line events-access
```

Comment:
disabled across all params childs
as they are being called by governance contract only 
and their before setting, we are emitting event

Reply:
https://github.com/crytic/slither/wiki/Detector-Documentation#missing-events-arithmetic. actually you need to mention check name which is event-maths